src/web/components/Container/Container.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Container from './';
+
+describe('Container', () => {
+  let content;
+  let wrapper;
+
+  beforeEach(() => {
+    content = <div id="test">Hello world</div>;
+    wrapper = shallow(<Container>{content}</Container>);
+  });
+
+  it('should render children component', () => {
+    const children = wrapper.find('div#test');
```

Comment:
Actually, you are testing that the `<div />` contains children.

I think you wanted to be sure that children exists and has at list one item.

If you want to test this, you should use:

```javascript
expect(wrapper.children().length).toBe(1)
```

Reply:
Ok thanks